src/Kafka/ConsumerGroups/components/ConsumerGroupResetOffset.tsx

```diff
@@ -0,0 +1,251 @@
+    clientId: `${t("consumerGroup.client_id")} + ${t(
+      "consumerGroup.member_id"
+    )}`,
+    current_offset: t("consumerGroup.current_offset"),
+    log_end_offset: t("consumerGroup.log_end_offset"),
+    offset_lag: t("consumerGroup.offset_lag"),
+    new_offset: t("consumerGroup.new_offset"),
+  };
+
+  return (
+    <Modal
+      variant={ModalVariant.large}
+      isOpen={isModalOpen}
+      aria-label={"Modal for resetting offset of Kafka consumergroup"}
+      title={t("consumerGroup.reset_offset")}
+      showClose={true}
+      aria-describedby="modal-message"
+      actions={[
+        <Button
+          variant="danger"
+          key={1}
+          isDisabled={isResetOffsetDisabled()}
+          onClick={onClickResetOffset}
+        >
+          {t("consumerGroup.reset_offset")}
+        </Button>,
+        <Button variant="link" key={2} onClick={onClickClose}>
+          {t("common:cancel")}
+        </Button>,
+      ]}
+    >
+      <Stack hasGutter>
+        <StackItem>
+          <Form isHorizontal>
+            <FormGroup label="Consumer group" fieldId="horizontal-form-name">
+              <Title className="form-title" headingLevel="h4" size="md">
+                {groupId}
+              </Title>
+            </FormGroup>
+            {isDisconnected && (
+              <FormGroup label="Topic" fieldId="horizontal-form-name">
+                <TopicDropdown
+                  value={selectedTopic}
+                  topics={topics}
+                  onChange={onChangeTopic}
+                />
+              </FormGroup>
+            )}
+            {isDisconnected && selectedTopic && (
+              <FormGroup label="New offset" fieldId="offset-dropdown">
+                <OffsetDropdown
+                  value={selectedOffset ? selectedOffset : t("common:select")}
+                  onChange={onChangeOffsetValue}
+                />
+              </FormGroup>
+            )}
+            {isDisconnected && selectedTopic && selectedOffset === "absolute" && (
+              <FormGroup label="Custom offset" fieldId="custom-offset-input">
+                <TextInput
+                  id="custom-offset-input"
+                  value={customOffsetValue}
+                  onChange={setcustomOffsetValue}
+                  type="number"
+                />
+              </FormGroup>
+            )}
+          </Form>
+        </StackItem>
+        <StackItem>
+          {!isDisconnected && (
+            <Alert
+              className="modal-alert"
+              variant="danger"
+              isInline
+              title={t("consumerGroup.reset_offset_connected_alert_title")}
+            >
+              <p>{t("consumerGroup.reset_offset_connected_alert_body")}</p>
+            </Alert>
+          )}
+        </StackItem>
+        <StackItem>
+          {isDisconnected && consumers.length > 0 && selectedTopic && (
+            <Stack hasGutter>
+              <StackItem>
+                <TableComposable
```

Comment:
Can we try using the `ResponsiveTable` component we have in the shared components for this?

Reply:
@riccardo-forina, I have a column for selecting the row in the table, but I don't think we can achieve that with the  `ResponsiveTable `

Reply:
It should be supported, the `Th` you get from the `ResponsiveTable` is HOC of the `Th` you use here. All its features are supported.

Did you try and get some errors?

Reply:
I need 
![image](https://user-images.githubusercontent.com/53568062/170674527-c88fe9c0-27b7-4211-af0f-d691f36fc480.png) selectable column, Not sure how to deal with this with the `ResponsiveTable`

Reply:
I added an example here. I'll update it to be more readable eventually, but it should give you an idea of how to do this.

https://github.com/redhat-developer/app-services-ui-components/pull/377

Reply:
As per your comment, https://github.com/redhat-developer/app-services-ui-components/pull/377#issuecomment-1139487866 says not to follow this approach, so for now can I use the normal table?

Reply:
Correct